### DISCUSSION TITLE
[Feature request] FAQ entry and documentation for small things with LXQT

### DISCUSSION THREAD

**Discussion by rubyFeedback** [lxqt/lxqt#2523] [Ideas]
Hey Lxqt folks,

It's been about two years since I last compiled lxqt from source. Right now
I go through each component, look for new releases, update this, then
compile it via some well-tested ruby scripts. So far this is fine.

However had, I don't recall how to start lxqt from init 3 in oldschool
linux systems (aka no systemd). Usually I just start it via "startx", but
via some "exec icewm" or something like that. I forgot how to do 
this for lxqt. It is probably super simple, but ... i was also thinking,
what if lxqt project would have a FAQ, where such simple questions
can be answered? Do you guys use a FAQ? If so, is there a link to it?

If not then this should be added IMO. Kind of as a substitute to 
stackoverflow, so we don't have to ask questions that are trivial
to answer. I don't use mailing lists and I suck at webforum these
days, even phpbb (and I hate discourse). Randomly googling
for things is my preferred way to add new content into my local
knowledge base, this is why I think a FAQ would be super useful.
People could work through it to learn something new.

So if there is no LXQT FAQ, it would be nice to add one; and link
to it too, such as "some questions may also be answered at our
official FAQ here INSERT_URL".

Anyway that's my request - please consider whether this may be
useful. It could also be hosted at the github wiki, though the
github wiki is not ideal. For a FAQ it may suffice though. (I have to
check on this lateron when I finished compiling; right now I am
focused on finishing the new components.)

**Comment by stefonarch** [comment]
> Usually I just start it via "startx", but
via some "exec icewm" or something like that. I forgot how to do
this for lxqt. It is probably super simple, but ...

https://github.com/lxqt/lxqt/wiki/ConfigLaunch#launching-from-virtual-terminals


>what if lxqt project would have a FAQ, where such simple questions
can be answered? Do you guys use a FAQ? If so, is there a link to it?

The wiki should cover all those questions, the issue is more that people don't find it IMO.
